client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -894,6 +897,27 @@ const Editor = (props: EditorProps): JSX.Element => {
     updateFile({ fileKey, editorValue, editableRegionBoundaries });
   };
 
+  function createBreadcrumb(): HTMLOListElement {
+    const { block, superBlock } = props;
+    const breadcrumb = document.createElement('ol');
```

Comment:
This should be wrapped in a nav element similar to the current implementation.

Reply:
It seems that I overlooked the semantic and accessibility roll of `nav` element and [its `aria-label` attribute](https://www.w3.org/WAI/tutorials/page-structure/labels/#using-aria-label), so I will try to wrap the breadcrumb list in the one.